src/main/java/org/stellar/base/AssetTypeNative.java

```diff
@@ -7,6 +7,16 @@
   public AssetTypeNative() {}
 
   @Override
+  public String getCode() {
```

Comment:
Could you perhaps describe the motivation behind adding these methods to the native asset?